lib/net/ssh/transport/algorithms.rb

```diff
@@ -197,8 +197,8 @@ def initialized?
 
         def host_key_format
           case host_key
-          when "ssh-rsa-cert-v01@openssh.com", "ssh-rsa-cert-v00@openssh.com"
-            "ssh-rsa"
+          when /^([a-z0-9-]+)-cert-v\d{2}@openssh.com$/
+            $1 # rubocop:disable Style/PerlBackrefs
```

Comment:
I'd say disable the PerlBackref check globally

Reply:
This could also be substituted for `$MATCH` to use the english variant.